project/Settings.scala

```diff
@@ -302,4 +319,8 @@ object Settings {
     }
   }
 
+  def emptyIfScala213(value: Option[String], scalaBinaryVersionString: String): Option[String] = {
```

Comment:
I am not completely sure how MiMa works when the version does not exist. Is this needed at all?

If so, then perhaps we can change the naming such that it is easy to enable MiMa for slick 3.4.x once slick 3.4.0 has been released.

Maybe we could do the following in the build.sbt
```scala
binaryCompatSlickVersion in ThisBuild := {
  if (scalaBinaryVersion.value.startsWith("2.13")) None else Some("3.3.0")
}
```

Reply:
I made the suggested change

Reply:
Thanks! One more thing this `def emptyIfScala213` is no longer needed